tests/sqlparser_common.rs

```diff
@@ -3354,8 +3354,13 @@ fn parse_create_table_clone() {
 
 #[test]
 fn parse_create_table_trailing_comma() {
-    let sql = "CREATE TABLE foo (bar int,)";
-    all_dialects().one_statement_parses_to(sql, "CREATE TABLE foo (bar INT)");
+    let dialect = TestedDialects {
```

Comment:
I am worried about this change -- I think it means that the trailing comma will no longer be supported by all dialects

Reply:
Note that all dialects allowed trailing commas only in create statements because create statements are parsed by a special function and not by the `parse_comma_separated` that actually considers the `trailing_commas` option. So that was a bug not a feature; however, as mentioned before, that behavior can be restored using the parsingOptions struct.

PS: I pointed that out in our previous conversation and recommended a follow-up to migrate the create statement to use the general `parse_comma_separated` function.